### PR TITLE
Eye Safe Pocket Blogger

### PR DESCRIPTION
**Pull request by notadroid1007**
## Describe your project in 1-3 sentences
A very cool idea to make a portable blogger, with multiple options, bluetooth, camera, mic, and a an 7 color E-Ink display to protect your eyes always :)

## Hack Club Slack handle
@NeeravGupta

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by anddddrew** [comment]
Hi @notadroid1007, this project looks great. Excited to see you build this

Let me get a 2nd pair of eyes on this and from there we can go forward :)

**Comment by BadPotato1007** [comment]
> Hi @notadroid1007, this project looks great. Excited to see you build this
> 
> Let me get a 2nd pair of eyes on this and from there we can go forward :)

Sure! Thank you for your support in this hardware fest!

**Comment by notadroid1007** [comment]
> Hi @notadroid1007, this project looks great. Excited to see you build this
> 
> Let me get a 2nd pair of eyes on this and from there we can go forward :)

Sure! waiting for the reply!

**Review by cfanoulis — COMMENTED** [review]
Hello Neerav! This sounds very useful! Just checking a couple of things before we hit the big green button:

- Are you positive (pun intended) that the battery can power the Pi? I didn't see any connectors in your budget
- Have you accounted for shipping&taxes?

thx!

**Comment by BadPotato1007** [comment]
> Hello Neerav! This sounds very useful! Just checking a couple of things before we hit the big green button:
> 
>     * Are you positive (pun intended) that the battery can power the Pi? I didn't see any connectors in your budget
> 
>     * Have you accounted for shipping&taxes?
> 
> 
> thx!

Yes, I am positive. Also I already have the connectors, and thebattry module can charge the battery, and use the adapter as the main power supply at the same time!

**Comment by bellesea** [comment]
Hi @notadroid1007! Thanks for your proposal. It's been almost a week since cf's message and we want to know if you're still interested in the grant. Please reply in the next 48 hours to this PR (or this email) if you're still interested, else we'll be closing this proposal. Thank you :)

**Comment by bellesea** [comment]
Hello! Unfortunately 2 days have passed, which means that I'll be closing this PR as stale. If you still wish to pursue a grant please let us know by replying to this PR before Jan 15th at 11:59pm ET.

**Comment by notadroid1007** [comment]
hey! I am so sorry! I was out of town, and could not reply! I would love to make this project, if you issue me the grant!

**Comment by cfanoulis** [comment]
hellooo! thanks for replying - i'm reopening this, please answer the following before eod today thanks :)
> 
    Are you positive (pun intended) that the battery can power the Pi? I didn't see any connectors in your budget
    Have you accounted for shipping&taxes?

**Comment by notadroid1007** [comment]
yes, I am positive that the battery can power the pi, as Ihave done it before, and I have the connectors, so I did not mention that in the bill!

**Comment by cfanoulis** [comment]
pls include those (& any sales tax) in your budget with a price of 0, thanks :) we need to keep an accurate bill of materials for each project

**Comment by notadroid1007** [comment]
> pls include those (& any sales tax) in your budget with a price of 0, thanks :) we need to keep an accurate bill of materials for each project

Done! please check it and let me know!

File: notadroid1007.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ These are the things I need to make my amazing project...
 | Battery connectors   | Owned                                                                                    | £ ---  |
 | Bluetooth Keyboard   | Owned                                                                                    | £ ---  |
 | Shipping             | ------------------------------- Shipping ----------------------------------------------- | £37.20 |
-| Total                |                                                                                          | £205.80|
+| Tax                  |                                                                                          | £4.10  |
+| Total                |                                                                                          | £209.90|
 
 
 ## £205.80 GBP is $246.57 USD
```
Comment (cfanoulis) [inline notadroid1007.md:55]:
```suggestion
## £209.90 GBP is $256.73 USD (approx.)
```

heyyyy thanks for including tax! after conversion you seem to be slightly above the 250USD grant limit - would you be able to cover the rest? thanks :)

**Review by cfanoulis — CHANGES_REQUESTED** [review]
I swear - final thing

**Comment by cfanoulis** [comment]
hey neerav? just noticed this is actually over 250$, sorry for that! can you please confirm you can pay the amount over 250$ for this project? thx <3

**Comment by cfanoulis** [comment]
@notadroid1007 see above^

**Comment by notadroid1007** [comment]
> hey neerav? just noticed this is actually over 250$, sorry for that! can you please confirm you can pay the amount over 250$ for this project? thx <3

hey! yes I am ready to pay the remaining amount from my own pocket :)

**Review by cfanoulis — APPROVED** [review]
perfect, tysm for bearing with me :) i can't wait to see you shipping this - remember, we're building the entire 10 days, so I'd love to maybe see some text-to-speech function for making your own audiobooks, like those narrators on some websites.

**Comment by cfanoulis** [comment]
@hackclub/bank-ops will be in touch shortly to get you setup on Hack Club Bank :)

**Review by MelSmith104 — COMMENTED** [review]
hey @notadroid1007 i love this idea! i really love how it ties into another interest of yours :) im dm'ing you on slack right now to chat about next steps!

**Comment by hackliv** [comment]
Neerav has decided to move forward with this project: https://github.com/hackclub/winter/pull/164 instead!

**Pull request closed without merging**